Formula/courier-unicode.rb

```diff
@@ -0,0 +1,43 @@
+class CourierUnicode < Formula
+  desc "Implements several algorithms related to the Unicode Standard"
+  homepage "http://www.courier-mta.org/unicode/"
+  url "https://downloads.sourceforge.net/project/courier/courier-unicode/1.4/courier-unicode-1.4.tar.bz2"
+  sha256 "2174f4cdd2cd3fe554d4cbbd9557abac0e54c0226084f368bcb2e66b0e78cf96"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+
+    # https://sourceforge.net/p/courier/mailman/message/34719502/
+    inreplace "courier-unicode.h", "extern const char *unicode_locale_chset_l(locale_t l);", ""
+    system "make"
+    system "make", "install"
+  end
+
+  # https://sourceforge.net/p/courier/mailman/message/34719502/
+  patch :p0, :DATA
```

Comment:
@MikeMcQuaid the project has moved to Github https://github.com/svarshavchik/courier-libs but I had issues attempting to use that source in a Homebrew formula. Additionally I couldn't get maildrop to compile on OSX and as reason for adding courier-libs was as a upstream dependency I figure we should pass until those issues are resolved